web_src/js/features/repo-projects.js

```diff
@@ -168,24 +182,27 @@ export function initRepoProject() {
     });
   });
 
-  $('#new_board_submit').on('click', function (e) {
+  $('#new_board_submit').on('click', (e) => {
     e.preventDefault();
-
     const boardTitle = $('#new_board');
     const projectColorInput = $('#new_board_color_picker');
+    if (boardTitle.val().length < 1) {
+      return false;
```

Comment:
There is already `preventDefault` above, so here can be a simple `return`

And the check can be simplified to `    if (!boardTitle.val()) {`